builtin/types/chainconfig/chainconfig.proto

```diff
@@ -100,8 +100,13 @@ message AppStoreConfig {
     uint64 num_evm_keys_to_prune = 1;
 }
 
+message EvmConfig {
+    uint64 gas_limit = 1;
+}
+
 message Config {
     AppStoreConfig app_store = 2;
+    EvmConfig evm = 3;
```

Comment:
Huh, how come the fields start at 2?

Reply:
well, good question

Reply:
To be safe, I make `EvmConfig = 1` because `AppStoreConfig` has been merged